frontend/src/OrganizationSettingsPage/index.jsx

```diff
@@ -18,7 +19,22 @@ export function OrganizationSettings(props) {
   const { updateSidebarNAV } = useContext(BreadCrumbContext);
   const { workspaceId } = useParams();
 
-  const sideBarNavs = ['Users', 'Groups', 'Workspace login', 'Workspace variables'];
+  const conditionObj = { admin: true, hide: false };
```

Comment:
admin value should be taken from 
const [admin, setAdmin] = useState(authenticationService.currentSessionValue?.admin);
this state

Reply:
hide is not needed